RetroAchievements/RAWeb
# fix(multiset): correctly update player count metrics on metrics updates

This PR:
* Updates `UpsertGameCoreAchievementSetFromLegacyFlags` to properly write `players_total` and `players_hardcore` metrics on game metrics updates. Currently, sets only have their player counts written on create, and they are never updated.
* Allows `ra:platform:game:update-metrics` to run across all games in the database.
* Introduces a `ra:platform:verify-achievement-set-integrity` command, which I'd like to run in prod before merging https://github.com/RetroAchievements/RAWeb/pull/2719.

The new command can be used to verify the data integrity of achievement sets in terms of their metrics and attached achievements. It can be used like so:
```
sail artisan ra:platform:verify-achievement-set-integrity 1 # sets for a single game
sail artisan ra:platform:verify-achievement-set-integrity # all games/sets
```

**How to test this PR:**
Make sure Horizon is running.
```bash
sail artisan ra:platform:verify-achievement-set-integrity 1

# an integrity issue should be logged in your terminal

sail artisan ra:platform:game:update-metrics 1
sail artisan ra:platform:verify-achievement-set-integrity 1

# no integrity issues

sail artisan ra:platform:verify-achievement-set-integrity

# lots of integrity issues should be reported!!

sail artisan ra:platform:game:update-metrics

# on my quite-fast machine, jobs take about 45 minutes to finish.

sail artisan ra:platform:verify-achievement-set-integrity

# no integrity issues
```
